modules/ztp-creating-siteconfig-custom-resources.adoc

```diff
@@ -71,8 +71,7 @@ spec:
    releaseImage: quay.io/openshift-release-dev/ocp-release:4.10.0-x86_64 <2>
 ----
 <1> The descriptive version that you want to deploy.
-<2> Points to the specific release image to deploy.
-
+<2> Specifies the `releaseImage` to deploy and determines the OS Image. The discovery ISO bases the releaseImage on the specified image or the latest version if the specified image is unavailable.
```

Comment:
Just the make it a clearer and more explicit, consider something like this rephrasing:
"Specifies the `releaseImage` to deploy and determines the version of OS image accordingly. I.e. The discovery ISO will be based on an OS image version as the specified `releaseImage`, or latest if the exact version is unavailable."

Hope it describes the service logic well enough, just a recap for the detailed behaviour:
* Extract the x.y.z version from `releaseImage` (e.g. 4.10.1)
* Search for OS image with the exact version x.y.z (e.g. 4.10.1)
* If not found, search for the latest z-stream (e.g. 4.10.z)
* Use the OS image to create the discovery ISO